pkg/appender/store.go

```diff
@@ -255,29 +255,34 @@ func (cs *chunkStore) chunkByTime(t int64, isVariantEncoding bool) *attrAppender
 		chunk := chunkenc.NewChunk(cs.logger, isVariantEncoding) // TODO: init based on schema, use init function
 		app, err := chunk.Appender()
 		if err != nil {
-			return nil
+			cs.logger.Error(err)
```

Comment:
you are returning here an err (where previously it wasn't returned), which might affect the flow in the condition at line 350.
so better understand why the error was ignored before

Reply:
it's הכנה למזגן, there's no Appender that actually returns an error